### DISCUSSION TITLE
disable video compression

### DISCUSSION THREAD

**Discussion by Snazzie** [rustdesk/rustdesk#4757] [General]
I noticed that no matter which mode i choose, quality, reaction time. there is always artifacts on the video feed. doesnt help taking bitrate to max in custom.

Im assuming its down to the compression algorithm used. Can this be solved by adding the ability to disable compression algorithm? or maybe instead of % slider for bitrate, allow us to enter a specific number

**Comment by melroy89** [comment]
I have the same issue.. I will now try to self host my own server and see if that helps.. But otherwise its not great 😢

**Reply by melroy89** [reply]
Although I also found you can select 'more' option. Which allows you to increase the bit rate I thought to like 9000%.

**Comment by xlionjuan** [comment]
Without codec you can't even capture the screen.

**Comment by southerndoc** [comment]
> I noticed that no matter which mode i choose, quality, reaction time. there is always artifacts on the video feed. doesnt help taking bitrate to max in custom.
> 
> Im assuming its down to the compression algorithm used. Can this be solved by adding the ability to disable compression algorithm? or maybe instead of % slider for bitrate, allow us to enter a specific number

I would love to see ability to disable video compression.

**Comment by southerndoc** [comment]
Thinking about this more, I'm not sure disabling the video compression is an option as it would eat up a lot of bandwidth.

I've found the sweet spot to be AV1 on M1 Macs.  For some reason H265 is a little laggy even with hardware codec enabled.
